src/XR/features/WebXRControllerPointerSelection.ts

```diff
@@ -366,9 +388,23 @@ export class WebXRControllerPointerSelection extends WebXRAbstractFeature {
                     scene.pointerY = this._screenCoordinatesRef.y;
                 }
             }
-            controllerData.pick = this._scene.pickWithRay(controllerData.tmpRay, this._scene.pointerMovePredicate || this.raySelectionPredicate);
-
-            const pick = controllerData.pick;
+            let pick = this._pickWithPoint(controllerData.pointForNearInteration);
+            let nearPick = pick && pick.pickedPoint && pick.hit;
+ 
+            if (nearPick) {
+                console.log("SUCCESS!");

```

Comment:
I'd be very happy to see that in my console., but some won't :-)

There other other console outputs as well

Reply:
Hi Raana, for sure. I am not ready for a review yet. We are still testing and i did not want to remove the console logs yet. This is supposed to be a draft PR, not sure what changed. Also, I just realized i made changes to a branch that is way behind the master. Sorry for the confusion. Let me start a fresh draft Pr. Please discard this.